backend/src/main/java/com/soen341/instagram/dto/AccountDTO.java

```diff
@@ -0,0 +1,86 @@
+package com.soen341.instagram.dto;
+
+import java.util.Date;
+
+import javax.validation.constraints.NotNull;
+
+public class AccountDTO
```

Comment:
Maybe call this `CreateAccountRequestDTO` or something along those lines? `AccountDTO` would be more appropriate for something like a `/getAccount` endpoint that would return the information of the specified user (and such an endpoint should not return the email, password, etc.)

Reply:
Also, perhaps create another package so that it becomes something like: `com.soen341.instagram.dto.account.AccountDTO.java`